backend/tracim_backend/config.py

```diff
@@ -156,6 +156,20 @@ def _get_protected_value(self, value: str, secret: bool) -> str:
             return value
 
 
+class IdPConfig(object):
+    def __init__(self, displayed_name: str, identifier: str, logo_url: str):
```

Comment:
need a description explaining that this is related to SAML / Shibbolet identification/authentication

Reply:
renamed it to SamLIdPConfig

Reply:
Why `SamLIdPConfig` and not `SAMLIdPConfig` (or `SamlIdpConfig`)?

Reply:
For readability.

SAMLIdPConfig is kind of hard to read and SamlIdPConfig is even worse on most fonts since lowercase l and uppercase I can look very similar.

No issue in changing it to a more conventional naming, but I'd prefer to keep it that way so it's not hard to read (at least for me)

Reply:
So, I suggest to put an inline comment: `# INFO - M.L - 2023-10-25: the naming SamLIdPConfig has been preferred to SAMLIdpConfig / SamlIdPConfig for readability`

This way, developers will not decide to change this naming in the future

Reply:
(and they will understand why)

Reply:
Done and noted !